src/network.c

```diff
@@ -7,6 +7,10 @@
 #include <sys/epoll.h>
 #include <sys/socket.h>
 
+enum {
+    MAX_SIZE_MESSAGE_INT = (1024*1024)/32+2
```

Comment:
```suggestion
    MAX_SIZE_MESSAGE_INT = ((1024*1024)/32)+2 // 1 Mib + 2 ints for overhead
```
just for readability